### PR TITLE
Update docs slightly

### PR DESCRIPTION
**Pull request by surminus**
Please see detail in commits.

### CODE REVIEW

File: doc/guides/styleguide.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+If the project contains a large number of outputs, include these in a separate `outputs.tf`
+file.
+
+3. Include a backend file named `<environment>.<stackname>.backend` (you may
+use the [create-backends tool](../../tools/create-backends.sh) to do this)
+
+4. Generate [`remote_state.tf`](../../tools/generate-remote-state-boiler-plate.sh)
+and [`user_data_snippets.tf`](../../tools/generate-user-data-boiler-plate.sh) files.
+
+5. Run `terraform fmt` across your manifests.
```
Comment (issyl0) [inline doc/guides/styleguide.md]: Is it worth mentioning about the [docs tool](https://github.com/alphagov/govuk-aws/blob/master/tools/update-docs.sh)? I always forget it exists.

Comment (issyl0) [reply doc/guides/styleguide.md]: Done in 5cb5fd6.

**Comment by surminus** [comment]
:+1: I can't approve, though

File: doc/guides/getting-started.md [hunk 2]
```diff
@@ -61,28 +59,7 @@ done by the GDS Verify team.
...
-2. Where possible, put variables and outputs within `main.tf`.
-
-3. Include a backend file named `<environment>.<stackname>.backend` (you may
-use the [create-backends tool](../../tools/create-backends.sh) to do this)
-
-4. Generate [`remote_state.tf`](../../tools/generate-remote-state-boiler-plate.sh)
-and [`user_data_snippets.tf`](../../tools/generate-user-data-boiler-plate.sh) files.
-
-5. Run `terraform fmt` across your manifests.
+Please follow the [styleguide](styleguides.md) when developing in the repository.
```
Comment (issyl0) [inline doc/guides/getting-started.md]: The filename is actually `styleguide.md` (singular).

**Pull request merged**